ansible.cfg

```diff
@@ -11,6 +11,9 @@ log_path = /var/log/ansible.log
 # Disable them in the context of https://review.openstack.org/#/c/469644
 retry_files_enabled = False
 
+# Option to retry failed ssh executions if the failure is encountered in ssh itself
+retries = 5
```

Comment:
According to the docs at https://docs.ansible.com/ansible/latest/intro_configuration.html#openssh-specific-settings this would need to go under the `[ssh_connection]` header